.editorconfig

```diff
@@ -149,7 +149,7 @@ dotnet_diagnostic.SA1202.severity = warning  # Elements should be ordered by acc
 dotnet_diagnostic.SA1203.severity = warning  # Constants should appear before fields
 dotnet_diagnostic.SA1204.severity = warning  # Static elements should appear before instance elements
 dotnet_diagnostic.SA1205.severity = warning  # Partial elements should declare access
-dotnet_diagnostic.SA1206.severity = warning  # Declaration keywords should follow order
+dotnet_diagnostic.SA1206.severity = none     # Declaration keywords should follow order
```

Comment:
a lot of change already. fair enough :) I will spend some time on it on a follow up PR (probably just add the disable pragmas wherever it's not happy)

Reply:
I spent a fair bit of time trying to get it to work, but I don't think it is possible as long as 6.0 is also a target.